compose/material3/material3-common/src/commonMain/kotlin/androidx/compose/material3/common/InteractiveComponentSize.kt

```diff
@@ -67,7 +69,8 @@ internal object MinimumInteractiveModifier : ModifierNodeElement<MinimumInteract
                 "interactions if the element would measure smaller"
     }
 
-    override fun hashCode(): Int = System.identityHashCode(this)
+    @OptIn(InternalComposeApi::class)
```

Comment:
It is not allowed to use internal APIs between modules. You must use the same approach like in material (expect/actual)